base/Dockerfile

```diff
@@ -20,8 +22,9 @@ RUN conda install --yes -c conda-forge nomkl cytoolz cmake mamba python=${PYTHON
     pandas \
     tini==0.18.0 \
     cachey \
-    streamz \
-    && mamba clean -tipsy \
+    streamz
+
+RUN mamba clean -tipsy \
```

Comment:
Just pointing out that I separated the cache clearing step from the python installs to hopefully make debugging easier in the future 🙂

Reply:
I think these were originally in a single `RUN` command for Docker caching related reasons (cc @jacobtomlinson). I'm curious how separating these lines makes debugging easier

Reply:
Yeah I wasn't sure if this would cause some issues that justified this all happening in a single `RUN` command - happy to switch it back over if this is the case.

> I'm curious how separating these lines makes debugging easier

It's pretty small, but this would make it so the error output from a failed Docker build would be slightly more granular, so that failures resulting from the conda environment solve and cache clearing are distinct from one another. That being said, I imagine failures stemming from the cache clearing are somewhat unlikely, so I'm okay with merging these two `RUN`s back together if needed

Reply:
I think it would be best to merge this back. It's pretty common practice with Docker containers to install packages and then clean up temp files in a single step. This avoids unnecessary file size from intermediate layers.

Reply:
Thanks for the feedback @jacobtomlinson - remerged the steps in both Dockerfiles